### DISCUSSION TITLE
How are Github Actions Minutes billed?

### DISCUSSION THREAD

**Discussion by okaufmann** [community/community#26247] [Actions]
<p>Hi,</p>
<p>We have a self hostet Github Action Runner to run e2e tests.</p>
<p>Does anyone know if the minutes an action run on a self-hosted Runner will also be counted to the free minutes of the account?</p>
<p>Thanks!</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/okaufmann">@okaufmann</a>, welcome to the GitHub community!</p>
<p><a href="https://docs.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners#differences-between-github-hosted-and-self-hosted-runners">Self hosted runners are free</a> and don’t count towards your usage minutes:</p>
<blockquote>
<p>Self-hosted runners are free to use with GitHub Actions, but you are responsible for the cost of maintaining your runner machines.</p>
</blockquote>

**Comment by okaufmann** [comment]
<p>Hi <a class="mention" href="/thomasshaped">@thomasshaped</a></p>
<p>Thank you! I totally missed that…</p>

**Comment by ghost** [comment]
<p>Hi <a class="mention" href="/thomasshaped">@ThomasShaped</a>  you mean If I have Private repositories already and then If I use self-hosted runners, no cost for git actions even beyond 50,000 minutes.</p>

**Comment by otter-computer** [comment]
<p>Hi <a class="mention" href="/sreeharinath-rangaraju_bay">@sreeharinath-rangaraju_bay</a>, that’s right! Self-hosted runners don’t count towards your minutes at all, so you can exceed 50000 minutes using them. You will be responsible for covering the operating costs of these runners with whatever hosting provider you host them on though.</p>
